geppetto.js/geppetto-ui/src/list-viewer/ListViewer.js

```diff
@@ -392,6 +392,16 @@ export default class ListViewer extends React.Component {
     );
   }
 }
+
+ListViewer.defaultProps = {
+  handler: {},
+  infiniteScroll: true,
+  columnConfiguration: [],
+  filter: () => {},
+  customComponents: {},
+  showPagination: false,
+}
+
 ListViewer.propTypes = {
```

Comment:
Something doesn't seem to be working properly with the listViewer now:
![image](https://user-images.githubusercontent.com/19196034/152330526-1be54e35-985d-438b-b22c-6962d3b88f7f.png)

I think the reason for this might be a couple of props types that mismatch with what is provided.
You can look for warnings like this to help you identify whose props are mismatching
![image](https://user-images.githubusercontent.com/19196034/152330851-19ebeaa1-c657-4c55-b5c9-74c38a575f07.png)

Let me know how it goes